### DISCUSSION TITLE
Creating portable case including the evidence files

### DISCUSSION THREAD

**Discussion by Chaoskeeper** [sepinf-inc/IPED#355] [Q&A]
Hi, 
working alot with IPED, but I'm missing one thing at the moment.

Is there a way create a portable case including the evidence source files (usually *.E01) where all links to the files are already correct.
so the result should be:
1) Create portable case with source
2) Give whole package to investigator
3) Investigator opens iped without having to relink when trying to get details on a file

thanks for any info

**Comment by lfcnassif** [comment]
I have just described it in the user manual: https://github.com/sepinf-inc/IPED/wiki/User-Manual#--portable

**Reply by Chaoskeeper** [reply]
Yeah I'm aware of this option. (Thanks anyway)
Was just wondering if my evidences are spread around the system, in very different locations due to internal rules

So when using iped, I'll be moving the evidence before using it.

Will work for me ;)

**Reply by lfcnassif** [reply]
Maybe creating soft links inside the case folder to evidences could help.
